rake/spec/default/60-st2_all-services-ok_spec.rb

```diff
@@ -72,7 +72,7 @@
 
 # all st2 services should work immediately after restart
 describe 'st2 services availability after restart' do
-  describe command("st2ctl restart && st2 action list") do
+  describe command("systemctl daemon-reload; st2ctl restart && st2 action list") do
```

Comment:
Can we add `systemctl daemon-reload` to postinst scriptlets?
And so this way the instruction won't be needed to be duplicated here or in any other installer like Ansilbe, Chef, st2ctl, etc.

Somewhat relevant https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=950726

Reply:
Also, see https://github.com/StackStorm/st2-packages/issues/495 which could be also closed if we add it now.